build_tools/requirements.txt

```diff
@@ -29,3 +29,7 @@ matplotlib==3.4.3; sys_platform == 'darwin'
 lxml
 pytools 
 cffi
+dominate
+html5lib
+importlib-resources
+importlib_resources
```

Comment:
Is there any difference between these two?

Reply:
Yes, one is python < 3.7, one is >= 3.7, I've included both for now in case people have python < 3.7.